src/wovnio/html/HtmlConverter.php

```diff
@@ -84,6 +100,37 @@ private function insertSnippet($dom) {
     }
   }
 
+  /**
+   * Insert hreflang tags for all supported_langs
+   *
+   * @param simple_html_dom_node $dom
+   */
+  public function insertHreflangTags($dom) {
+    $lang_codes = $this->store->settings['supported_langs'];
```

Comment:
$lang_codes should be `$store->settings['supported_langs']` + `iso639_1Normalization` lang code (for Chinese) imo